helm/alfresco-content-services/7.2.N_values.yaml

```diff
@@ -41,9 +41,9 @@ share:
 alfresco-search:
   enabled: true
   searchServicesImage:
-    tag: 2.0.3.6
+    tag: 2.0.12
```

Comment:
https://docs.alfresco.com/content-services/7.2/support/ I checked the doc it is set to 2.0.3 is it outdated?